commands/apps.go

```diff
@@ -273,13 +274,16 @@ func RunAppsCreate(c *CmdConfig) error {
 		return err
 	}
 
+	var errs error
+
 	if wait {
 		apps := c.Apps()
 		notice("App creation is in progress, waiting for app to be running")
 		err := waitForActiveDeployment(apps, app.ID, "")
 		if err != nil {
-			warn("App deployment couldn't enter `running` state: %v", err)
-			return c.Display(displayers.Apps{app})
+			errs = multierror.Append(errs, fmt.Errorf("app deployment couldn't enter `running` state: %v", err))
+			errs = multierror.Append(errs, c.Display(displayers.Apps{app}))
+			return errs
 		} else {
 			app, _ = c.Apps().Get(app.ID)
 		}
```

Comment:
```suggestion
			errs = multierror.Append(errs, fmt.Errorf("app deployment couldn't enter `running` state: %v", err))
			if err := c.Display(displayers.Apps{app}); err != nil {
				errs = multierror.Append(errs, err)
			}

			return errs
		}

		app, _ = c.Apps().Get(app.ID)
```

Seems like multierror doesn't mind having a nil error appended, but probably a good practice to check anyways. As we're here, we could also get rid of the unnecessary else statement. (Applies to the other uses below too.)

Reply:
thanks for the suggestion!